doc/source/_toc.yml

```diff
@@ -2,8 +2,13 @@ format: jb-book
 root: index
 parts:
   - caption: Ray
-    chapters:
+    chapters: 
       - file: ray-overview/index
+        title: "Overview"
+        sections:
+          - file: ray-overview/workloads
+            title: "Common ML workloads"
```

Comment:
<img width="259" alt="image" src="https://user-images.githubusercontent.com/4529381/218892393-dc743f29-14f7-4f0a-a2af-fe204698a9ce.png">
this ends up rendering a bit oddly (it wasn't clear that the header was clickable)

Reply:
You mean the carrot didn't show up?

Reply:
no, i mean that it wasnt clear to me that you could actually click the overview page

Reply:
Ah. Yes, this is a source of debate, whether we should put the "Common ML workloads" page at the top level. Max was trying to keep the top level short because it was really bloated. Are you suggesting that I pull it up to the top level?

Reply:
Maybe what's not clear is that the `Overview` page links to the `Common ML workloads` page: 
<img width="723" alt="Screenshot 2023-02-14 at 5 36 31 PM" src="https://user-images.githubusercontent.com/122562471/218905207-5125b683-7679-46ec-b2df-f94b226f2780.png">

Reply:
I pulled the Workloads page to the top level in the side nav.
<img width="260" alt="Screenshot 2023-02-14 at 6 38 56 PM" src="https://user-images.githubusercontent.com/122562471/218913322-05529083-f733-43f3-b70e-74c43bf65703.png">